acceptance/suites/tests/code_commands/code_scripts.rb

```diff
@@ -210,6 +217,6 @@ class { '::ntp': }
   key       ="/etc/puppetlabs/puppet/ssl/private_keys/#{fqdn}.pem"
   hostcert  ="/etc/puppetlabs/puppet/ssl/certs/#{fqdn}.pem"
   auth_str  ="--cacert #{cacert} --key #{key} --cert #{hostcert}"
-  result=on(master, "curl --silent #{auth_str} https://#{fqdn}:8140/puppet/v3/catalog/#{fqdn}?environment=production | python -m json.tool").stdout
+  result=on(master, "curl --silent #{auth_str} https://#{fqdn}:8140/puppet/v3/catalog/#{fqdn}?environment=production | #{python_bin} -m json.tool").stdout
```

Comment:
Is this line needed? I thought `python -m json.tool` was used to pretty print a string of valid json. I would assume that `JSON.parse` could handle an un-pretty printed string of JSON though...

It doesn't look like we print the result anywhere humans would have to read it either.

Reply:
As I recall, the result of `on( )` calls are logged by beaker when `--debug` is on.

So, this is likely here for those moments when a human is trawling through the output trying to figure out why the test failed.

Reply:
fair